# This file is part of https://github.com/KurtBoehm/tlaxcaltin.
#
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at https://mozilla.org/MPL/2.0/.

diff --git a/src/libscotchmetis/library_parmetis.h b/src/libscotchmetis/library_parmetis.h
index 2ca4f1d..950ecd4 100644
--- a/src/libscotchmetis/library_parmetis.h
+++ b/src/libscotchmetis/library_parmetis.h
@@ -71,6 +71,7 @@
 #define __parmetis_h__
 
 #include <mpi.h>                                  /* Since ParMeTiS does it, do it too */
+#include <metis.h>
 
 #endif /* __parmetis_h__ */
 
